sass/responsive/_mobile.scss

```diff
@@ -1586,9 +1590,13 @@
                 .SidebarChannel {
                     height: 45px;
 
-                    a,
-                    button {
+                    .SidebarLink {
                         height: 45px;
+                        align-items: center;
+
```

Comment:
Where is the display: flex? :D adding it should align vertically!

Reply:
https://github.com/mattermost/mattermost-webapp/blob/feature/experimental_sidebar_features-v2/sass/layout/_sidebar-left.scss#L893

Right here!

Reply:
Oh weird